thruster-core/Cargo.toml

```diff
@@ -28,6 +28,6 @@ serde_derive = "1.0.24"
 smallvec = "0.6.2"
 templatify = "0.2.3"
 time = "0.1"
-tokio = { version = "=0.2.5", branch = "master" }
-tokio-util = { version = "=0.2.0", branch = "master", features = ["full"]  }
+tokio = { version = "0.2" }
```

Comment:
If we could keep these locked to exact versions, that would be awesome! 

In general I like to keep versions locked to avoid accidental breaking changes in minor version changes in dependencies (I've seen it happen! Especially in the npm realm.) In addition, this makes it very explicit what dependency this package uses without another developer needing to look through lockfiles to figure it out.

Reply:
I am afraid that this approach is generally incompatible with semver and the Rust ecosystem in general. It forces downstream users to use this specific version as well with no availability to upgrade. Then imagine what happens if you have two crates depending upon tokio, each locked to a specific minor release.

FWIW [Tokio does consider minor version breakages as bugs and fixes them promptly](https://github.com/tokio-rs/tokio/pull/1991).

Reply:
That's a great point -- didn't think of downstream breakage! Disregard this and the other dittos!